dojo/tools/trufflehog/parser.py

```diff
@@ -193,7 +193,8 @@ def get_findings_v3(self, data, test):
                     url="N/A",
                     dynamic_finding=False,
                     static_finding=True,
-                    nb_occurences=1
+                    nb_occurences=1,
+                    payload=rawV2
```

Comment:
Are you sending the secrets in clear ?

Reply:
In-transit encryption can be set from scanners to DefectDojo instance (and as a security solution I suppose that every organization has done it)